lib/ast-converter.js

```diff
@@ -954,8 +954,14 @@ module.exports = function(ast, extra) {
                 break;
 
             case SyntaxKind.VariableStatement:
+                var variableDeclarationType = "VariableDeclaration",
+                    variableIsAmbient = ts.isInAmbientContext(node);
+
+                if (variableIsAmbient) {
+                    variableDeclarationType = "TSAmbientVariableDefinition";
```

Comment:
I think this should be TSAmbientVariableDeclaration since it fits better with estree. What was the reason behind VariableDefinition?